src/general-components/API/calls/Saves.tsx

```diff
@@ -181,11 +190,45 @@ const lockSave = async (saveID: number, lock: boolean, apiArgs?: APIArgs) => {
 /**
  * Erstellt einen neuen Save
  *
+ * @param name
+ * @param description
+ * @param tool_id
  * @param data Daten des Save
+ * @param resources
  * @param apiArgs API Argumente
  */
-const createSave = async <D extends unknown>(data: FormData, apiArgs?: APIArgs) => {
-    return await callAPI<SaveResource<D>>("api/saves", "POST", data, true, apiArgs);
+const createSave = async <D extends object>(
+    name: string,
+    description: string,
+    tool_id: number,
+    data: D,
+    resources: ImportJSONData[],
+    apiArgs?: APIArgs
+) => {
+    let formData = new FormData();
+    formData.set("name", name);
+    formData.set("description", description);
+    formData.set("tool_id", tool_id.toString());
+    formData.set("data", JSON.stringify(data));
+
+    // Ressourcen
+    let i = 0;
+    for (const resource of resources) {
+        let res = await fetch(resource.file);
+        let blob = new File(
+            [new Uint8Array(await res.arrayBuffer())],
+            resource.name,
```

Comment:
The name of the file is ignored by the backend since the api changes. As a result it would be possible to pass the original file object wo the formdata